pmm/vbox-staging-start.groovy

```diff
@@ -119,155 +140,78 @@ pipeline {
-                                sudo yum -y install pmm-client --enablerepo=percona-testing-*
-                            else
-                                wget --progress=dot:giga "https://www.percona.com/downloads/pmm-client/pmm-client-\${CLIENT_VERSION}/binary/tarball/pmm-client-\${CLIENT_VERSION}.tar.gz"
-                                tar -zxpf pmm-client-\${CLIENT_VERSION}.tar.gz
-                                pushd pmm-client-\${CLIENT_VERSION}
-                                    sudo ./install
-                                popd
-                            fi
-
-                            sleep 10
-                            docker logs \${VM_NAME}-server
-
-                            export PATH=\$PATH:/usr/sbin
-                            sudo pmm-admin config --client-name pmm-client-hostname --server \$IP
-                        "
-                    """
-                }
-            }
-        }
-
-        stage('Run Clients') {
-            steps {
-                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
-                    sh """
-                        [ -z "${CLIENTS}" ] && exit 0 || :
-                        ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@\$(cat IP) "
-                            set -o errexit
-                            set -o xtrace
-
-                            export PATH=\$PATH:/usr/sbin
-                            test -f /usr/lib64/libsasl2.so.2 || sudo ln -s /usr/lib64/libsasl2.so.3.0.0 /usr/lib64/libsasl2.so.2
-
-                            bash /srv/percona-qa/pmm-tests/pmm-framework.sh \
-                                --pxc-version ${PXC_VERSION} \
-                                --ps-version  ${PS_VERSION} \
-                                --ms-version  ${MS_VERSION} \
-                                --md-version  ${MD_VERSION} \
-                                --mo-version  ${MO_VERSION} \
-                                --download \
-                                ${CLIENTS} \
-                                --sysbench-data-load \
-                                --sysbench-oltp-run
-                        "
-                    """
-                }
+                    ssh -i "/tmp/${env.VM_NAME}/sshkey" -o ConnectTimeout=1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null admin@${env.IP} "
+                        set -o errexit
+                        set -o xtrace
+                        sudo pmm-admin config --server ${env.IP}
+                        sudo service mysqld start
+                        sleep 10
+                        sudo service mysqld status
+                        sudo pmm-admin add mysql ms1 --host localhost
+                        sudo yum -y install wget svn cyrus-sasl-devel
+                        ls -lh /usr/lib64/libsasl*
```

Comment:
Is it really needed? or was added just for debug ?

Reply:
yes was added for debugging, will remove it..thanks :+1: